### ISSUE TITLE
cooler vs clodius

### ISSUE THREAD

**Issue by keiranmraine** [higlass/higlass-server#106]
The `requirements.txt` file specifies a fixed version of cooler (0.7.11), but indicates `clodius>=0.10.3`.  This results in the following warning:

```
ERROR: clodius 0.10.8 has requirement cooler>=0.8.0, but you'll have cooler 0.7.11 which is incompatible.
```

Would it not be safer to use `pip freeze` to fix all versions at a known good version for the release?

**Comment by pkerpedjiev** [comment]
Yes, yes it would. I think @flekschas just created a PR for that here: https://github.com/higlass/higlass-server/pull/107/files

There'll be a new release over the weekend with that change.

**Comment by flekschas** [comment]
Yes I fixed/pinned all versions in #107

**Comment by pkerpedjiev** [comment]
Just released a new server version: v1.10.3 should have this issue fixed.
